docs/metrics.md

```diff
@@ -0,0 +1,136 @@
+# Send Metrics
+The metrics collection and analysis plan for Send, a forthcoming Test Pilot experiment.
+
+## Analysis
+Data collected by Send will be used to answer the following high-level questions:
+
+- How do users
```

Comment:
Is this bullet missing words? Other top level list items are questions, this is... some existential, ominous foreshadowing.